presto-product-tests/src/main/java/com/facebook/presto/tests/utils/QueryExecutors.java

```diff
@@ -34,5 +34,10 @@ public static QueryExecutor onHive()
         return testContext().getDependency(QueryExecutor.class, "hive");
     }
 
+    public static QueryExecutor onSqlServer()
+    {
+        return testContext().getDependency(QueryExecutor.class, "sqlserver");
```

Comment:
Is this commit useful by itself? I'm thinking it should be merged with one of the other product-tests commits.